pwa/src/components/deleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import "./deletemodal.css";
+import { Modal } from "@conductionnl/nl-design-system";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+interface DeleteModalProps {
+  handleDelete: any;
+  handleId: any;
+  optionalLength?: any;
+}
+
+const DeleteModal: React.FC<DeleteModalProps> = ({ handleDelete, handleId, optionalLength }) => {
+  return (
+    <div className="DeleteModal">
+      <Modal
+        title="Warning!"
+        id={`deleteModal${handleId}`}
+        body={function () {
+          return (
+            <>
+              {optionalLength === 1 ? (
+                <p>
+                  Are you sure you want to delete {handleId}? <br></br>
+                  <b>If you delete this translation you wil also delete the Translation Table!</b>
```

Comment:
This is an important one to understand (if something's unclear about this comment, call me, we can go over it 👍🏼 )

Right now, you're adding logic within the `DeleteModal` based on where it is implemented (`TranslationTable`). So, you're writing logic for outside context. A couple of issues arise here:

1. The `DeleteModal` gets more complexity (especially for maintenance, you understand why you did this, other people might not);
2. When we have another edge case, we have to add more of this complex logic to the `DeleteModal`.

What we want is for any component (in this case the `DeleteModal`) so be completely unaware of its context (so the `DeleteModal` shouldn't care that it's being implemented inside a `TranslationTable`).